shiny/Elicitation/DESCRIPTION

```diff
@@ -1,9 +1,15 @@
 Type: Shiny
 Title: Expert Prior Elicitation
 License: FreeBSD + file LICENSE
-Author: Mike Dietze <dietze@bu.edu> 
-Author: Rob Kooper <kooper@illinois.edu>
+Author:
+	Mike Dietze <dietze@bu.edu>,
+	Rob Kooper <kooper@illinois.edu>
```

Comment:
👍 for better formatting! For future reference, I think we might want to move to the slightly more formal (or at least `pkgdown`-friendly) form:

```
Authors@R: c(person("Mike", "Dietze", email = "dietze@bu.edu"), person(first = "Rob", last = "Kooper", email = "kooper@illinois.edu"))
```

However, since the Shiny apps aren't being `pkgdown`ed, might not be necessary here.

Reply:
I approve of this in principle but haven't yet written an authors@R and decided I'd learned enough new things for one PR 😉